sources/Storages/Database/Database.php

```diff
@@ -0,0 +1,13 @@
+<?php
+namespace Ciebit\Users\Storages\Database;
+
+use Ciebit\Users\User;
+use Ciebit\Users\Collection;
+use Ciebit\Users\Storages\Storage;
+
+interface Database extends Storage
+{
+    public function setTableGet(string $name): self;
```

Comment:
Pode resumir esse método para:

```suggestion
    public function setTable(string $name): self;
```
A proposta que tinha feito em módulos anteriores se mostrou inviável.